test/package.json

```diff
@@ -28,13 +28,11 @@
     "@bedrock/jsonld-document-loader": "^4.0.0",
     "@bedrock/kms": "^14.0.0",
     "@bedrock/kms-http": "^18.0.0",
-    "@bedrock/ledger-context": "^24.0.0",
```

Comment:
This is a required peer dependency of `@bedrock/veres-one-context` so it should not be removed.